coverage/lcov-report/src/controllers/index.html

```
@@ -23,9 +23,9 @@ <h1><a href="../../index.html">All files</a> src/controllers</h1>
         <div class='clearfix'>
             
             <div class='fl pad1y space-right2'>
-                <span class="strong">41.14% </span>
+                <span class="strong">40.66% </span>
                 <span class="quiet">Statements</span>
-                <span class='fraction'>237/576</span>
+                <span class='fraction'>244/600</span>
```

HTMLHint (1.5.0)

[error] The value of attribute [ class ] must be in double quotes.

(attr-value-double-quotes)